container-registry/aws/ecr/versions.tf

```diff
@@ -1,13 +1,24 @@
+provider "skopeo2" {
+  destination {
+    login_username = data.aws_ecr_authorization_token.current.user_name
+    login_password = data.aws_ecr_authorization_token.current.password
```

Comment:
Is it possible to put this in the skopeo resource? if so, I think it would make the deployment more reliable as the configuration of the provider would not depend on AWS resource.

Reply:
the way of doing this is deprecated and will be removed in the next realease